components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ExportUtils.java

```diff
@@ -872,10 +873,12 @@ public static void addAPIMetaInformationToArchive(String archivePath, APIDTO api
                                 exportFormat, ImportExportConstants.GRAPHQL_COMPLEXITY, graphQLQueryComplexityInfoDTO);
                     }
                 }
+                API api = APIMappingUtil.fromDTOtoAPI(apiDtoToReturn, apiDtoToReturn.getProvider());
+                String organizationId = ApiMgtDAO.getInstance().getOrganizationIDByAPIUUID(apiIdentifier.getUUID());
```

Comment:
`apiIdentifier.getUUID()` might sometimes be null. Instead we will have to retrieve UUID from identifier via a separate call.

Reply:
Or can we get it from the API object in line 876?